design/general-progress-monitoring.md

```diff
@@ -0,0 +1,462 @@
+# Backup and Restore Item Action Progress Monitoring
+
+This is intended as a replacement for the previously-approved Upload Progress Monitoring design
+([Upload Progress Monitoring](upload-progress.md)) in order to expand the supported use cases beyond
+snapshot uploads to include what was previously called Async Backup/Restore Item Actions. This
+updated design should handle the combined set of use cases for those previously separate designs.
+
+Volume snapshotter plug-in are used by Velero to take snapshots of persistent volume contents. 
```

Comment:
Maybe BackupItemAction should be mentioned here, because CSI plugin uses BIA to take snapshot.